test/infrastructure/docker/go.mod

```diff
@@ -11,15 +11,11 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20191120175047-4206685974f2
 	k8s.io/api v0.17.2
 	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
+	k8s.io/client-go v11.0.0+incompatible
```

Comment:
Any particular reason not to prefer `v0.17.2` to make it clear it aligns with the version of other k8s packages imported?

Reply:
This is the usual go mod updating stuff without asking, reverted